helix-loader/src/grammar.rs

```diff
@@ -89,15 +89,106 @@ pub fn fetch_grammars() -> Result<()> {
     let mut grammars = get_grammar_configs()?;
     grammars.retain(|grammar| !matches!(grammar.source, GrammarSource::Local { .. }));
 
-    run_parallel(grammars, fetch_grammar, "fetch")
+    print!("Fetching {} grammars", grammars.len());
+    let results = run_parallel(grammars, fetch_grammar);
+    println!(" ... done");
+
+    let mut errors = Vec::new();
+    let mut git_updated = Vec::new();
+    let mut git_up_to_date = Vec::new();
+    let mut non_git = Vec::new();
+
+    for res in results {
+        match res {
+            Ok(FetchSuccess {
+                grammar_id,
+                action: FetchAction::GitUpdated { revision },
+            }) => git_updated.push(format!("{}->{}", grammar_id, revision)),
+            Ok(FetchSuccess {
+                grammar_id,
+                action: FetchAction::GitUpToDate,
+            }) => git_up_to_date.push(grammar_id),
```

Comment:
It looks like only `Vec::len` is used so this could be a usize that gets incremented instead

Reply:
Ah yes, I originally had them displayed but decided that if it hasn't changed, it's not very useful